infogami/infobase/server.py

```diff
@@ -94,8 +94,8 @@ def g(self, *a, **kw):
             else:
                 process_exception(e)
 
-        
-        result = d.json_data if isinstance(d, JSON) else json.dumps(d)
+        # use default=str to deal with datetimes on Python 3
+        result = d.json_data if isinstance(d, JSON) else json.dumps(d, default=str)
```

Comment:
This should have errored on python2 as well; why is this necessary?

![image](https://user-images.githubusercontent.com/6251786/88219334-6d72f100-cc2f-11ea-8334-21a47bf2ed5a.png)

Reply:
Removed these changes.  Will attack in a separate PR.